src/rgw/rgw_op.cc

```diff
@@ -3997,6 +3997,13 @@ void RGWPutMetadataObject::execute()
   }
 
   rgw_get_request_metadata(s->cct, s->info, attrs);
+
+  if (!requested_placement_id.empty() &&
```

Comment:
Hmm, I'm not 100% sure we really need to handle `X-Storage-Policy` on **objects**. In [the related SwiftStack's documentation](https://www.swiftstack.com/docs/admin/cluster_management/policies.html) we can find:
> Swift clients select the Storage Policy that will be used when storing object data under a container's namespace when they create the container. All object data stored under a container's namespace will be placed according the configuration of the Storage Policy which was set when the container was created.

In the Swift object server's code base I don't see any traces of the handling. What does Swift do when different `X-Storage-Policy` is seen in `POST` on object?